apps/govern/components/Docs/Docs.tsx

```diff
@@ -67,7 +67,7 @@ const WhatIsOlasGovern = () => (
         It allows to direct OLAS token emissions to desirable staking programs via a voting-weight
         mechanism.
       </li>
-      <li>It provides access to Olas DAO proposals on Boardroom and Snapshot.</li>
```

Comment:
removed both as we don't use Boardroom anymore and have the proposals page in menu anyway